### ISSUE TITLE
Publication year and doc year outputting as the same on Discover Citations

### ISSUE THREAD

**Issue by aoifespenge** [wellcometrust/reach#438]
Spotted by Alex: After a couple of different search terms it seems like the Publication Year and Document year are same, when this is unlikely the case, as usually it takes many years to get cited. It looks like the ones that are showing up are from that publication year but the research publications that are form other years aren't showing up. 
<img width="640" alt="Screen Shot 2020-02-18 at 14 54 15" src="https://user-images.githubusercontent.com/25910370/74747483-f049e700-525e-11ea-87df-97dfa20b56e4.png">

**Comment by SamDepardieu** [comment]
Good catch, we're also quite unlikely to have that data as it's not even in the policy-docs list

**Comment by dd207** [comment]
How about removing 'Publication Year' from the policy document results?
